test/unittests/eof_validation_test.cpp

```diff
@@ -310,8 +312,15 @@ TEST(eof_validation, EOF1_invalid_section_0_type)
 
 TEST(eof_validation, EOF1_too_many_code_sections)
 {
-    const auto valid = "EF0001 011000" + bytecode{"020400"} + 0x400 * bytecode{"0001"} +
-                       "040000 00" + 0x400 * bytecode{"00800000"} + 0x400 * bytecode{"FE"};
+    std::string code_sections_1024;
+    for (int i = 0; i < 1024; ++i)
+        if (i < 1023)
+            code_sections_1024 += "E5" + hex(big_endian(static_cast<uint16_t>(i + 1)));
+        else
+            code_sections_1024 += "5B5B00";
+
+    const auto valid = "EF0001 011000" + bytecode{"020400"} + 0x400 * bytecode{"0003"} +
+                       "040000 00" + 0x400 * bytecode{"00800000"} + code_sections_1024;
     EXPECT_EQ(validate_eof(valid), EOFValidationError::success);
 
     const auto invalid = "EF0001 011002" + bytecode{"020401"} + 0x401 * bytecode{"0001"} +
```

Comment:
It would make ssense to use `code_sections_1024` in invalid case, too, just adding one more after them.

Reply:
I added another variable `code_sections_1025` in order that section 1024 is calling section 1025 (so we don't have unreachable code sections).